CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+### Removed
+
+- Remove validation of `location` field in `AzureMachine` CRs.
```

Comment:
Maybe worth mentioning the change that uses the location passed as parameter to the admission controller rather than the value in the CR.

Reply:
added